src/theme.js

```diff
@@ -0,0 +1,89 @@
+import { createTheme } from "@mui/material/styles";
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: "#1a73e8", // Google blue
+    },
+    secondary: {
+      main: "#db4437", // Google red
+    },
+    warning: {
+      main: "#fbbc05", // Google yellow
+    },
+    success: {
+      main: "#34a853", // Google green
+    },
+    common: {
+      black: "#000000",
+      white: "#FFFFFF",
+      grey: {
+        50: "#FAFAFA",
+        100: "#F6F6F6",
+        200: "#EAEAEA",
+        300: "#D4D4D4",
+        400: "#C4C4C4",
+        500: "#959595",
+        600: "#666666",
+        700: "#525252",
+        800: "#333333",
+      },
+    },
+    eventBackground: "#F6F6F6",
+    teamsBackground: "#EAEAEA",
+  },
+  typography: {
+    fontFamily: "Roboto",
```

Comment:
Hmm 🤔 we still have Roboto 

```suggestion
    fontFamily: "Open Sans",
```